src/components/MachineSelector/tests/mockData.ts

```diff
@@ -0,0 +1,3 @@
+export const mockMachines = {
```

Comment:
we will have a lot of mock values for a lot of stuff, what was the discussion regarding using some package to generate these mock values?

Reply:
We decided not to use extra package rather make a separate file inside each test directory named mockData